### ISSUE TITLE
Check logic of if

### ISSUE THREAD

**Issue by deniscostadsc** [twstrike/libotrv4#107]
## Why

During boolean refactoring (#96), I had change this line code [from this](https://github.com/otrv4/libotr-ng/blob/2500b9c7b088c150da27b748c482f5593197de6c/src/otrng.c#L2065) [to this](https://github.com/otrv4/libotr-ng/blob/a802f56f979765fac9eb4fd62e8e2aefe509fbe4/src/otrng.c#L2065) on [this commit](a802f56f979765fac9eb4fd62e8e2aefe509fbe4). This has changed the logic and this need to be checked to avoid future bugs.

before: https://github.com/otrv4/libotr-ng/blob/2500b9c7b088c150da27b748c482f5593197de6c/src/otrng.c#L2065
after: https://github.com/otrv4/libotr-ng/blob/a802f56f979765fac9eb4fd62e8e2aefe509fbe4/src/otrng.c#L2065

## Tasks
- [x] Add tests to reach that part of the code.
- [x] Check the coverage for that part

**Comment by claucece** [comment]
Mmmm.. I'm not sure why create a separate issue for this.. this is working now, right?

**Comment by deniscostadsc** [comment]
We already talked about this, but I'll make the discussion public.

The only guarantee I have it's working is that tests are now passing. Maybe it worthy take a look on this. If people disagree, we can just close the issue.

**Comment by claucece** [comment]
I've tested this and is working fine :) I'm closing this. Thanks for reporting it.
